### PR TITLE
Changes the guards and royal spawnpoints to the throne room (Dun Manor)

### PR DESCRIPTION
**Pull request by bowlofcereal**
## About The Pull Request

This has been requested a few times to me and I decided that it would probably be a good idea. On Dun Manor a lot of the time the garrison doesn't immediately report for orders due to them not spawning at the Keep. This should relatively fix that issue by returning most spawnpoints (The King, Queen, their children, the Hand, the Garrison, the Squires, the Captain, and the Veteran) to the throne room, so at roundstart they are all in there for immediate orders.

image

## Why It's Good For The Game

Bringing it in line with every other map- this is done everywhere but here. It makes thematic sense that everyone be together in one room at the start so everyone is fully aware of who eachother is.

I've not included the Court Magician, the Archivist, the Steward, or the Serving Staff as they are not as directly important to such orders. They will still have to make their way to the court.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

### CODE REVIEW

**Comment by bowlofcereal** [comment]
Tested it. No direct inherent issues.

**Comment by sanshoom** [comment]
Can you put the jesters spawnpoint ontop of the chandelier?

**Comment by Optimism333** [comment]
I was under the impression that the changes in #852 were done to center the Veteran more around the village. Putting them back in the throne room on round start seems a little counterintuitive. The rest of the garrison spawning there seems reasonable, I'm not sure about that change though.

**Review by Optimism333 — COMMENTED** [review]
I was under the impression that the changes in https://github.com/Darkrp-community/OpenKeep/pull/852 and others were done to center the Veteran more around the village. Putting them back in the throne room on round start seems a little counterintuitive. The rest of the garrison spawning there seems reasonable, I'm not sure about that change though.

**Comment by sanshoom** [comment]
> I was under the impression that the changes in #852 and others were done to center the Veteran more around the village. Putting them back in the throne room on round start seems a little counterintuitive. The rest of the garrison spawning there seems reasonable, I'm not sure about that change though.

>veteran player spawns in
>immediately goes into the town instead of the village

thats how it works

**Comment by bowlofcereal** [comment]
> I was under the impression that the changes in #852 and others were done to center the Veteran more around the village. Putting them back in the throne room on round start seems a little counterintuitive. The rest of the garrison spawning there seems reasonable, I'm not sure about that change though.

The veteran normally goes right to the court roundstart anyways. They're still a village extension, they are just spawning there with the rest of the guard. Its simply so they can receive orders like the other maps.

**Comment by Tipy1802** [comment]
Honestly, why not put the Court Mage, Steward and servants there while you are at it? It would save them time walking to the court to introduce themselves and receive order, give the king an overview of his courtiers and be consistent with the other maps

**Comment by NPC1314** [comment]
Lose vet, add servants, steward, court mage and any other court scum missing.

**Comment by NPC1314** [comment]
Can you summarize the roles this will place in courtroom please and lets roll it past the focus group to check if theres an angle we missed.

**Comment by bowlofcereal** [comment]
I'm not dealing with the map conflicts caused by this taking so long, and someone else said they'll do it for me.
